SampleApps/TestApp/AppDelegate.swift

```diff
@@ -10,28 +10,36 @@
 // governing permissions and limitations under the License.
 //
 
-import AEPAssurance
 import AEPCore
-import AEPEdge
-import AEPEdgeConsent
 import AEPEdgeIdentity
 import AEPServices
 import Compression
 import UIKit
 
+// MARK: TODO remove this once Assurance has tvOS support.
+#if os(iOS)
+import AEPAssurance
+import AEPEdge
+import AEPEdgeConsent
+#endif
+
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
     // TODO: Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
     private let ENVIRONMENT_FILE_ID = ""
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        var extensions = [Identity.self]
+        // MARK: TODO remove this once Assurance has tvOS support.
+        #if os(iOS)
+        extensions.append(Assurance.self)
+        extensions.append(Consent.self)
+        extensions.append(Edge.self)
```

Comment:
should we create a task to revisit these once they are available in a stable branch?

Reply:
Yes we will have to update all the edge extensions.